Patterns/Chinese/Chinese-NumbersWithUnit.yaml

```diff
@@ -684,13 +684,33 @@ DimensionSuffixList: !dictionary
     Pound: 磅
     Ounce: 盎司
 # Information
-    Bit: 比特|位|b
-    Byte: 字节|byte
-    Kilobyte: 千字节|kb
-    Megabyte: 兆字节|mb
-    Gigabyte: 十亿字节|千兆字节|gb
-    Terabyte: 万亿字节|兆兆字节|tb
-    Petabyte: 千兆兆|千万亿字节|pb
+    Bit: 比特|位|b|bit
+    Kilobit: 千比特|千位|kb|Kb
+    Megabyte: 兆比特|兆位|mb|Mb
+    Gigabyte: 十亿比特|千兆比特|十亿位|千兆位|gb|Gb
+    Terabyte: 万亿比特|兆兆比特|万亿位|兆兆位|tb|Tb
+    Petabyte: 千兆兆比特|千万亿比特|千兆兆位|千万亿位|pb|Pb
+    Byte: 字节|byte|Byte
+    Kilobyte: 千字节|kB|KB
+    Megabyte: 兆字节|mB|MB
+    Gigabyte: 十亿字节|千兆字节|gB|GB
+    Terabyte: 万亿字节|兆兆字节|tB|TB
+    Petabyte: 千兆兆字节|千万亿字节|pB|PB
+# Power
```

Comment:
For now it's better to comment out these Power/Frequency/Energy units here, as they are currently out of scope for the recognizers.
Don't delete them from the YAML file, just mark each line as a comment with '#'.